### ISSUE TITLE
Output file not working in 1.2.3 and 2.0.0-beta

### ISSUE THREAD

**Issue by zigomir** [sass/node-sass#621]
I'm running `node-sass` like so

```sh
node-sass --watch app/editor/styles app/editor/styles/app.scss public/app.css
```

and output file (`app.css) will be created in CWD instead of inside `public/` directory.

Version `1.2.2` works fine though.

**Comment by am11** [comment]
Hi, the output needs to be specified with `--output` option (alias: `-o`), like so:

```bash
node-sass --watch app/editor/styles -o public/ app/editor/styles/app.scss 
```

Note that the output is `output directory`, not the filename (which IMO shouldn't be the case).

**Comment by zigomir** [comment]
Hi, thank you! :) then looks like the way program works has changed a bit from version `1.2.2`.

**Comment by am11** [comment]
You are right, I was mistaken. The [code](https://github.com/sass/node-sass/blob/5ab23fb34c09d1135b7ab701d165f4d28fa7cedd/bin/node-sass#L130-L138) seems to handle the last argument as output file. It should work with target file without -o or --output options, which are meant to specify output dir. :)

I don't have time right away to test what's going wrong there. If you can, please try to debug (and test with rearranging the formation of command)? 

Apparently we don't have any test (in [cli test suit](https://github.com/sass/node-sass/blob/master/test/cli.js)) for the scenario where we are providing destination as a second parameter. 

Separately, If you don't specify the filename, it would generate a css file with same name as source scss at cwd path.

In any case, I am reopening this issue until we are confirm that the option is working properly.

**Comment by am11** [comment]
Side note, [this comment](https://github.com/sass/node-sass/blob/master/bin/node-sass#L135) is misleading. It should say `with respect to dir` (where dir is `options.output` if -o is set, otherwise `process.cwd()`).

**Comment by am11** [comment]
I have attempted to fix couple of related issues in 3bc8e22. 

Can you try applying this patch to `bin/node-sass` and let me know if this fixes your issue?
